src/components/Contributions.js

```diff
@@ -26,13 +26,17 @@ function Contributions({repoName, owner}) {
             <AccentLink href={`https://github.com/${owner}/${repoName}/contribute`}>make new contributions.</AccentLink>
           </MicroFont>
         </CardPadding>
-        <hr width="100%" />
         <List>
           {issues &&
             issues.map(issue => (
               <li key={issue.node.id}>
-                <a target="_blank" href={issue.node.url}>
-                  <IssuesListItem title={issue.node.title} labels={issue.node.labels} />
+                <a style={{textDecoration: "none"}} target="_blank" href={issue.node.url}>
```

Comment:
let's try to avoid adding inline styles and opt for adding className instead. The preference is really creating styled-components, but if that is too big of a lift then a className and some CSS in the top-level styled-component will be best.